authentik/stages/email/models.py

```diff
@@ -16,6 +16,9 @@
 from authentik.lib.config import CONFIG
 from authentik.lib.utils.time import timedelta_string_validator
 
+EMAIL_RECOVERY_MAX_ATTEMPTS = CONFIG.get_int("email_recovery.max_attempts")
```

Comment:
The default value can be hardcoded, it doesn't need to be adjustable globally as it can be set per-stage